protoc-gen-grpc-gateway/main.go

```diff
@@ -66,7 +66,9 @@ func main() {
 	}
 
 	reg.SetPrefix(*importPrefix)
-	reg.Load(req)
+	if err := reg.Load(req); err != nil {
+		emitError(err)
```

Comment:
@shurcooL It is intended.
https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/plugin.proto#L82